src/rgw/rgw_appmain.cc

```diff
@@ -536,6 +535,8 @@ void rgw::AppMain::init_lua()
   }
 #endif
 
+  env.lua.manager = env.store->get_lua_manager();
```

Comment:
why was this moved from line 411 to here?

Reply:
i just thought that `init_lua()` was a better place than `init_frontends2()`

Reply:
this needs to be tested with zone reloads.
the lua manager is used to access the system objects.